build/azure-pipelines-nightly.yml

```diff
@@ -6,248 +6,235 @@ trigger: none
-   
-- task: EsrpCodeSigning@1
-  displayName: 'ESRP CodeSigning Nuget Packages'
-  inputs:
-    ConnectedServiceName: 'ESRP CodeSigning - OData'
-    FolderPath: '$(Build.ArtifactStagingDirectory)\Nuget'
-    Pattern: '*.nupkg'
-    signConfigType: inlineSignParams
-    inlineOperation: |
-     [
-         {
-             "keyCode": "CP-401405",
-             "operationSetCode": "NuGetSign",
-             "parameters": [ ],
-             "toolName": "sign",
-             "toolVersion": "1.0"
-         },
-         {
-             "keyCode": "CP-401405",
-             "operationSetCode": "NuGetVerify",
-             "parameters": [ ],
-             "toolName": "sign",
-             "toolVersion": "1.0"
-         }
-     ]
-    VerboseLogin: true
-
-- task: PublishBuildArtifacts@1
-  displayName: 'Publish Artifact - Nuget Packages'
-  inputs:
-    PathtoPublish: '$(Build.ArtifactStagingDirectory)\Nuget'
-    ArtifactName: Nuget
-
-# Push the odata model builder nightly to MyGet
-- task: NuGetCommand@2
-  displayName: 'NuGet push - Nightly packages to MyGet'
-  inputs:
-    command: push
-    packagesToPush: '$(Build.ArtifactStagingDirectory)\Nuget\*Nightly*.nupkg'
-    nuGetFeedType: external
-    publishFeedCredentials: 'MyGet.org - OData.net new nightly feed'
\ No newline at end of file
+
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1ESPipelineTemplates
+  parameters:
+    pool:
+      name: MSSecurity-1ES-Build-Agents-Pool
+      image: MSSecurity-1ES-Windows-2019
+      os: windows
+    customBuildTags:
+    - ES365AIMigrationTooling
+    stages:
+    - stage: build
+      jobs:
+      - job: Release
+        templateContext:
+          outputs:
+          - output: pipelineArtifact
+            displayName: 'Publish Artifact - Nuget Packages'
+            targetPath: '$(Build.ArtifactStagingDirectory)\Nuget'
+            artifactName: Nuget
+          - output: nuget
+            displayName: 'NuGet push - Nightly packages to MyGet'
+            packageParentPath: '$(Build.ArtifactStagingDirectory)'
+            packagesToPush: '$(Build.ArtifactStagingDirectory)\Nuget\*Nightly*.nupkg'
+            nuGetFeedType: external
+            publishFeedCredentials: 'MyGet.org - ODL Feed'
+        steps:
+        - task: NuGetToolInstaller@0
+          inputs:
+            versionSpec: '>=5.2.0'
+
+        - task: UseDotNet@2
+          displayName: Use .NET 6.0
```

Comment:
should this be .net 8.0?

Reply:
@ElizabethOkerio I'll update that after I merge #41. This pull request is targeting current `main` that was target .NET 6.0